schemas/com.snowplowanalytics.badrows/enrichment_failures/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
My hope was that for any enrichment failures we would have the ability to query all the fields in the event that were not generated at enrichment. Have I misunderstood? Or is that not the case?

Reply:
So to clarify - if the payload is the tracker protocol + collector metadata then querying the individual fields should be relatively straightforward.